pipelines/tekton/build-container-image-pipeline/build-container-image-pipeline.yaml

```diff
@@ -19,7 +19,37 @@ spec:
     type: string
   - name: containerfileRelativePath
     type: string
+  - name: modelRelativePath
+    type: string
+  - name: fetch-model
+    type: string
+  - name: git-model-repo
+    type: string
+    default: ""
+  - name: git-revision
```

Comment:
Some parameter names are `camelCase` while others are `kebab-case` should we try make these consistent?